README.md

```diff
@@ -7,10 +7,107 @@ The firmware runs on the Synthstrom Audible Deluge’s Renesas RZ/A1L processor,
+
+### DBT Build Targets
+
+The following doesn't represent all possible build command variations but a sampling. What goes for building also goes for cleaning, so 
+
+`all` or `build` - Build for all supported targets (dbt-build-release-oled, dbt-build-release-7seg, dbt-build-debug-oled, and dbt-build-debug-7seg). Built files output to appropriately named subdirectories prefixed with `dbt-build-`.
+
+#### Build variables:
+
+* `OLED` - narrow the scope to only build targets for OLED or only build targets for 7segment displays. When left out, all display types are built. This variable can be stacked with other variables and applies to `clean` as well.
+    * eg: `./dbt all OLED=1` (build all **OLED** targets)
+    * eg: `./dbt all OLED=0` (build all **7segment** targets)
+* `DEBUG` - narrow the scope to only build targets with debugging support (including debugging symbols and different optimizations) or only release targets. When left out, both debugging and release targets are built. This variable can be stacked with other variables and applies to `clean` as well.
+    * eg: `./dbt all DEBUG=1` (build only **debugging** targets)
+    * eg: `./dbt all DEBUG=0` (build only **release** targets)
+
+#### Build arguments: 
+
+* `-c` or `--clean` - this is the built-in SCons clean option. When used with `./dbt all` it's equivalent to `./dbt clean`.
+* `--e2_orig_prefix` - this turns on support for the prefixes e2 Studio expects based on the original Deluge source tree. All instances of `dbt-build-` will be replaced with `e2-build-` and the output directories will change as well.
+* `--nowo`, `--no-owo`, or `--no-uwu` - DBT features a progress indicator (not always visible) in the form of an ASCII emote face that switches rapidly between OwO, UwU, etc. If this is too noisy in the logs, it can be turned off by adding one of these arguments.
+* `--verbose=[0-3]` - This controls the noise level of DBT and its underlying calls:
+    * `0` - Bare minimum. No notices, hidden warnings, and only critical errors are shown.
+    * `1` - Clean. Hidden warnings, and build steps are described without showing the actual command.
+    * `2` - Commands and warnings are visible.
+    * `3` - Everything is shown where possible.
+* `--spew` - This is shorthand for `--verbose=3` and `--nowo`, each described above.
+
+`clean` - Clean up all supported targets (dbt-build-release-oled, dbt-build-release-7seg, dbt-build-debug-oled, and dbt-build-debug-7seg)
+* Variables described above under `all` also apply to `clean`.
+
+*There are often multiple ways to do the same thing.* For instance the above targeting commands are shorthand for longer commands with specific targets, for example `./dbt all OLED=0 DEBUG=1` is equivalent to `./dbt dbt-build-debug-7seg`. Additionally, `./dbt clean` is equivalent to `./dbt all -c`, and so on.
+
+Additional detail on working with DBT will be coming soon.
```

Comment:
As of right now it barely does anything, so yeah internal plumbing in case it's needed in future. The original purpose was in case I needed to inject the version string via a templated DBT.h file, but I backtracked on that approach in favour of DBT owning the build.